functions/stabo/colsog_fn_dropStabo.sqf

```diff
@@ -43,4 +43,8 @@ _parentHelicopter setVariable ["COLSOG_staboRope", _staboRope, true];
 	0,																// Priority
 	false,															// Remove on completion (removes for allPlayers !)
 	false															// Show in unconscious state
-] remoteExec ["BIS_fnc_holdActionAdd", 0]; // no need for JIP parameter in remoteExec
\ No newline at end of file
+] remoteExec ["BIS_fnc_holdActionAdd", 0]; // no need for JIP parameter in remoteExec
+
+//There needs to be a sleep here, otherwise the physics of the rope falling causes some latent movement in the sandbag.
+sleep 1;
+_droppedSandbag enableSimulation false;
```

Comment:
This will not work because if sandbag is attached to rope, it will still move...